src/NServiceBus.Msmq.AcceptanceTests/When_saga_has_no_mapped_messages.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Support;
+    using NUnit.Framework;
+
+    public class When_saga_has_no_mapped_messages
```

Comment:
Would it be easier to make this a unit test instead? 

Similar to https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core.Tests/Sagas/When_saga_has_no_start_message.cs